sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -460,4 +685,28 @@ private EpidemiologicalData collectSpecificData() {
                     RESIDING_OR_TRAVELING_DETAILS_KNOWN_OPTIONS)))
         .build();
   }
+
+  private void selectAllActivityTypes() {
+    for (TypeOfActivityExposure value : TypeOfActivityExposure.values())
```

Comment:
we can create a list with valid values for "alive" situation in the enum class, so we can get straight a list with needed values, to avoid adding these conditions.